dart/sync/lib/src/interfaces.dart

```diff
@@ -23,7 +23,7 @@ abstract class PageLoader {
   bool get useShadowDom;
   PageLoaderElement get globalContext;
 
-  Object getInstance(Type type, [dynamic context]);
```

Comment:
Why the change? Although not directly addressed, I believe that https://www.dartlang.org/effective-dart/design/#do-annotate-with-object-instead-of-dynamic-to-indicate-any-object-is-accepted is the closest to a recommendation for this case.